ihaskell-display/ihaskell-widgets/ihaskell-widgets.cabal

```diff
@@ -110,10 +138,7 @@ library
 
 					 -- The singletons package version is locked to the compiler
 					 -- so let cabal choose the right one.
-                     , singletons -any
-
-  if impl (ghc >= 9.0)
-    build-depends:     singletons-base -any
+                     , singletons < 3.0.0
```

Comment:
I think this section didn't rebase correctly, can you add what we have in `master` so it works on GHC 9.0?